src/main/resources/db/changeLog.xml

```diff
@@ -73,4 +73,12 @@
             </column>
         </createTable>
     </changeSet>
+
+    <changeSet id="survey_status" author="Ville Juutila">
```

Comment:
Since it's not nullable property, should we add a query to assign some status to existing surveys?

Reply:
Resolved.